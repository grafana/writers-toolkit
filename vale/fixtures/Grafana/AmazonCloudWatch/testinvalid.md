Don't use:

- AWS CloudWatch
- cloudwatch
- Cloudwatch
- cloudWatch
