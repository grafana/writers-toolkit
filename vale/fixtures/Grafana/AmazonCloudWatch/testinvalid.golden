testinvalid.md:3:3:Grafana.AmazonCloudWatch:Use 'Amazon CloudWatch' instead of 'AWS CloudWatch'.
testinvalid.md:4:3:Grafana.AmazonCloudWatch:Use 'CloudWatch' instead of 'cloudwatch'.
testinvalid.md:5:3:Grafana.AmazonCloudWatch:Use 'CloudWatch' instead of 'Cloudwatch'.
testinvalid.md:6:3:Grafana.AmazonCloudWatch:Use 'CloudWatch' instead of 'cloudWatch'.
